src/arkode/arkode_lsrkstep.c

```diff
@@ -380,13 +378,11 @@ int lsrkStep_Init(ARKodeMem ark_mem, int init_type)
 
      ARK_FULLRHS_OTHER -> called elsewhere (e.g. for dense output)
 
-  If this function is called in ARK_FULLRHS_START or ARK_FULLRHS_END mode and
-  evaluating the RHS functions is necessary, we store the vector f(t,y) in Fe
-  for reuse in the first stage of the subsequent time step.
-
-  In ARK_FULLRHS_END mode we check if the method is "stiffly accurate" and, if
-  appropriate, copy the vector F[stages - 1] to F[0] for reuse in the first
-  stage of the subsequent time step.
+  If this function is called in ARK_FULLRHS_START the RHS function is always
+  evaluated.
+  In ARK_FULLRHS_END mode we check is adaptive step sizes are being used
+  and copy the RHS evaluation using in computing the embedding at the end of
+  the time step.
```

Comment:
```suggestion

  In ARK_FULLRHS_END mode we evaluate the RHS if an SSP method is being
  used otherwise we copy the RHS evaluation from the end of the STS step.
```